lib/httpotion.ex

```diff
@@ -251,13 +253,13 @@ defmodule HTTPotion.Base do
             %HTTPotion.Response{
               status_code: process_status_code(status_code),
               headers: process_response_headers(headers),
-              body: process_response_body(body)
+              body: process_response_body(headers, body)
```

Comment:
The headers were just processed with `process_response_headers(headers)`, I'm not sure raw headers should be used here. Maybe extract `process_response_headers(headers)` into a variable and use it both in the returned `headers:` and in the argument to this?

Reply:
Ahhh good call!